### ISSUE TITLE
Exception Message:** module 'cv2' has no attribute 'VideoCapture'

### ISSUE THREAD

**Issue by kkget** [Kosinkadink/ComfyUI-VideoHelperSuite#298]
# ComfyUI Error Report
## Error Details
- **Node Type:** VHS_LoadVideo
- **Exception Type:** AttributeError
- **Exception Message:** module 'cv2' has no attribute 'VideoCapture'
## Stack Trace
```
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\execution.py", line 323, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\execution.py", line 198, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\execution.py", line 169, in _map_node_over_list
    process_inputs(input_dict, i)

  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\execution.py", line 158, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 271, in load_video
    return load_video_cv(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^

  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 143, in load_video_cv
    (width, height, fps, duration, total_frames, target_frame_time) = next(gen)
                                                                      ^^^^^^^^^

  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 42, in cv_frame_generator
    video_cap = cv2.VideoCapture(strip_path(video))
                ^^^^^^^^^^^^^^^^

```
## System Information
- **ComfyUI Version:** v0.2.2-44-g56e8f5e
- **Arguments:** ComfyUI\main.py --windows-standalone-build
- **OS:** nt
- **Python Version:** 3.11.6 (tags/v3.11.6:8b6ee5b, Oct  2 2023, 14:57:12) [MSC v.1935 64 bit (AMD64)]
- **Embedded Python:** true
- **PyTorch Version:** 2.4.1+cu121
## Devices

- **Name:** cuda:0 NVIDIA GeForce RTX 3090 : cudaMallocAsync
  - **Type:** cuda
  - **VRAM Total:** 25769279488
  - **VRAM Free:** 24438112256
  - **Torch VRAM Total:** 0
  - **Torch VRAM Free:** 0

## Logs
```
2024-10-03 11:41:32,644 - root - INFO - Total VRAM 24576 MB, total RAM 32541 MB
2024-10-03 11:41:32,644 - root - INFO - pytorch version: 2.4.1+cu121
2024-10-03 11:41:32,648 - xformers - WARNING - WARNING[XFORMERS]: xFormers can't load C++/CUDA extensions. xFormers was built for:
    PyTorch 2.0.1+cu118 with CUDA 1108 (you have 2.4.1+cu121)
    Python  3.11.4 (you have 3.11.6)
  Please reinstall xformers (see https://github.com/facebookresearch/xformers#installing-xformers)
  Memory-efficient attention, SwiGLU, sparse and more won't be available.
  Set XFORMERS_MORE_DETAILS=1 for more details
2024-10-03 11:41:32,661 - root - INFO - xformers version: 0.0.21
2024-10-03 11:41:32,661 - root - INFO - Set vram state to: NORMAL_VRAM
2024-10-03 11:41:32,661 - root - INFO - Device: cuda:0 NVIDIA GeForce RTX 3090 : cudaMallocAsync
2024-10-03 11:41:33,634 - root - INFO - Using pytorch cross attention
2024-10-03 11:41:34,992 - root - INFO - [Prompt Server] web root: E:\AComfyUI\ComfyUI_windows\ComfyUI\web
2024-10-03 11:41:34,994 - root - INFO - Adding extra search path checkpoints E:\sd-aki\sd-webui-aki-v4.9\models/Stable-diffusion
2024-10-03 11:41:34,995 - root - INFO - Adding extra search path configs E:\sd-aki\sd-webui-aki-v4.9\models/Stable-diffusion
2024-10-03 11:41:34,996 - root - INFO - Adding extra search path vae E:\sd-aki\sd-webui-aki-v4.9\models/VAE
2024-10-03 11:41:34,996 - root - INFO - Adding extra search path loras E:\sd-aki\sd-webui-aki-v4.9\models/Lora
2024-10-03 11:41:34,996 - root - INFO - Adding extra search path loras E:\sd-aki\sd-webui-aki-v4.9\models/LyCORIS
2024-10-03 11:41:34,996 - root - INFO - Adding extra search path upscale_models E:\sd-aki\sd-webui-aki-v4.9\models/ESRGAN
2024-10-03 11:41:34,996 - root - INFO - Adding extra search path upscale_models E:\sd-aki\sd-webui-aki-v4.9\models/RealESRGAN
2024-10-03 11:41:34,996 - root - INFO - Adding extra search path upscale_models E:\sd-aki\sd-webui-aki-v4.9\models/SwinIR
2024-10-03 11:41:34,996 - root - INFO - Adding extra search path embeddings E:\sd-aki\sd-webui-aki-v4.9\embeddings
2024-10-03 11:41:34,996 - root - INFO - Adding extra search path hypernetworks E:\sd-aki\sd-webui-aki-v4.9\models/hypernetworks
2024-10-03 11:41:34,996 - root - INFO - Adding extra search path controlnet E:\sd-aki\sd-webui-aki-v4.9\extensions\sd-webui-controlnet\models
2024-10-03 11:41:38,639 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Allor\__init__.py", line 7, in <module>
    loader.check_updates()
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Allor\Loader.py", line 208, in check_updates
    repo.remotes.origin.fetch()
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\git\remote.py", line 1014, in fetch
    res = self._get_fetch_info_from_stderr(proc, progress, kill_after_timeout=kill_after_timeout)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\git\remote.py", line 853, in _get_fetch_info_from_stderr
    proc.wait(stderr=stderr_text)
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\git\cmd.py", line 600, in wait
    raise GitCommandError(remove_password_if_present(self.args), status, errstr)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git fetch -v -- origin
  stderr: 'fatal: unable to access 'https://github.com/Nourepide/ComfyUI-Allor.git/': Failed to connect to 127.0.0.1 port 31181: Connection refused'

2024-10-03 11:41:38,639 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Allor module for custom nodes: Cmd('git') failed due to: exit code(128)
  cmdline: git fetch -v -- origin
  stderr: 'fatal: unable to access 'https://github.com/Nourepide/ComfyUI-Allor.git/': Failed to connect to 127.0.0.1 port 31181: Connection refused'
2024-10-03 11:41:40,054 - root - ERROR - [Impact Pack] Failed to import due to several dependencies are missing!!!!
2024-10-03 11:41:40,760 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-BiRefNet\__init__.py", line 3, in <module>
    from .BiRefNet_node import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-BiRefNet\BiRefNet_node.py", line 7, in <module>
    from models.baseline import BiRefNet
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-BiRefNet\models\baseline.py", line 11, in <module>
    from models.backbones.build_backbone import build_backbone
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-BiRefNet\models\backbones\build_backbone.py", line 8, in <module>
    from utils import check_download_model
ImportError: cannot import name 'check_download_model' from 'utils' (E:\AComfyUI\ComfyUI_windows\ComfyUI\utils\__init__.py)

2024-10-03 11:41:40,760 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-BiRefNet module for custom nodes: cannot import name 'check_download_model' from 'utils' (E:\AComfyUI\ComfyUI_windows\ComfyUI\utils\__init__.py)
2024-10-03 11:41:40,764 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-BiRefNet-ZHO\__init__.py", line 1, in <module>
    from . import birefnet
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-BiRefNet-ZHO\birefnet.py", line 7, in <module>
    from models.baseline import BiRefNet
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-BiRefNet\models\baseline.py", line 11, in <module>
    from models.backbones.build_backbone import build_backbone
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-BiRefNet\models\backbones\build_backbone.py", line 8, in <module>
    from utils import check_download_model
ImportError: cannot import name 'check_download_model' from 'utils' (E:\AComfyUI\ComfyUI_windows\ComfyUI\utils\__init__.py)

2024-10-03 11:41:40,764 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-BiRefNet-ZHO module for custom nodes: cannot import name 'check_download_model' from 'utils' (E:\AComfyUI\ComfyUI_windows\ComfyUI\utils\__init__.py)
2024-10-03 11:41:40,914 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 936, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1073, in get_code
  File "<frozen importlib._bootstrap_external>", line 1130, in get_data
FileNotFoundError: [Errno 2] No such file or directory: 'E:\\AComfyUI\\ComfyUI_windows\\ComfyUI\\custom_nodes\\ComfyUI-CLIPSeg\\__init__.py'

2024-10-03 11:41:40,915 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-CLIPSeg module for custom nodes: [Errno 2] No such file or directory: 'E:\\AComfyUI\\ComfyUI_windows\\ComfyUI\\custom_nodes\\ComfyUI-CLIPSeg\\__init__.py'
2024-10-03 11:41:44,763 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 936, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1073, in get_code
  File "<frozen importlib._bootstrap_external>", line 1130, in get_data
FileNotFoundError: [Errno 2] No such file or directory: 'E:\\AComfyUI\\ComfyUI_windows\\ComfyUI\\custom_nodes\\comfyui-ext\\__init__.py'

2024-10-03 11:41:44,763 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui-ext module for custom nodes: [Errno 2] No such file or directory: 'E:\\AComfyUI\\ComfyUI_windows\\ComfyUI\\custom_nodes\\comfyui-ext\\__init__.py'
2024-10-03 11:41:45,199 - root - INFO - Total VRAM 24576 MB, total RAM 32541 MB
2024-10-03 11:41:45,201 - root - INFO - pytorch version: 2.4.1+cu121
2024-10-03 11:41:45,202 - root - INFO - xformers version: 0.0.21
2024-10-03 11:41:45,202 - root - INFO - Set vram state to: NORMAL_VRAM
2024-10-03 11:41:45,202 - root - INFO - Device: cuda:0 NVIDIA GeForce RTX 3090 : cudaMallocAsync
2024-10-03 11:41:45,225 - root - ERROR - [Impact Pack] Failed to import due to several dependencies are missing!!!!
2024-10-03 11:41:45,228 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\__init__.py", line 53, in <module>
    raise e
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\__init__.py", line 31, in <module>
    import impact.subpack_nodes  # This import must be done before cv2.
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\impact_subpack\impact\subpack_nodes.py", line 4, in <module>
    import impact.subcore as subcore
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\impact_subpack\impact\subcore.py", line 15, in <module>
    raise e
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\impact_subpack\impact\subcore.py", line 11, in <module>
    from ultralytics import YOLO
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\ultralytics\__init__.py", line 5, in <module>
    from ultralytics.data.explorer.explorer import Explorer
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\ultralytics\data\__init__.py", line 3, in <module>
    from .base import BaseDataset
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\ultralytics\data\base.py", line 17, in <module>
    from ultralytics.utils import DEFAULT_CFG, LOCAL_RANK, LOGGER, NUM_THREADS, TQDM
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\ultralytics\utils\__init__.py", line 108, in <module>
    cv2.setNumThreads(0)  # prevent OpenCV from multithreading (incompatible with PyTorch DataLoader)
    ^^^^^^^^^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'setNumThreads'

2024-10-03 11:41:45,228 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Impact-Pack module for custom nodes: module 'cv2' has no attribute 'setNumThreads'
2024-10-03 11:41:45,991 - numexpr.utils - INFO - Note: NumExpr detected 20 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.
2024-10-03 11:41:45,991 - numexpr.utils - INFO - NumExpr defaulting to 8 threads.
2024-10-03 11:41:46,222 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ\__init__.py", line 1, in <module>
    from .nodes import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ\nodes.py", line 18, in <module>
    from .liveportrait.live_portrait_pipeline import LivePortraitPipeline
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ\liveportrait\live_portrait_pipeline.py", line 16, in <module>
    from .utils.filter import smooth
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ\liveportrait\utils\filter.py", line 3, in <module>
    from pykalman import KalmanFilter
ModuleNotFoundError: No module named 'pykalman'

2024-10-03 11:41:46,222 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ module for custom nodes: No module named 'pykalman'
2024-10-03 11:41:47,526 - root - INFO - --------------
2024-10-03 11:41:47,526 - root - INFO - [91m ### Mixlab Nodes: [93mLoaded
2024-10-03 11:41:47,526 - root - INFO - ChatGPT.available True
2024-10-03 11:41:47,527 - root - INFO - edit_mask.available True
2024-10-03 11:41:47,532 - root - INFO - LaMaInpainting.available False
2024-10-03 11:41:47,700 - root - INFO - ClipInterrogator.available True
2024-10-03 11:41:47,870 - root - INFO - PromptGenerate.available True
2024-10-03 11:41:47,870 - root - INFO - ChinesePrompt.available True
2024-10-03 11:41:47,871 - root - INFO - RembgNode_.available False
2024-10-03 11:41:48,382 - root - INFO - TripoSR.available
2024-10-03 11:41:48,383 - root - INFO - MiniCPMNode.available
2024-10-03 11:41:48,617 - root - INFO - Scenedetect.available False
2024-10-03 11:41:48,710 - root - INFO - FishSpeech.available False
2024-10-03 11:41:48,713 - root - INFO - SenseVoice.available False
2024-10-03 11:41:48,714 - root - INFO - [93m -------------- [0m
2024-10-03 11:41:58,009 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui-reactor-node\__init__.py", line 23, in <module>
    from .nodes import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui-reactor-node\nodes.py", line 15, in <module>
    from insightface.app.common import Face
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\insightface\__init__.py", line 18, in <module>
    from . import app
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\insightface\app\__init__.py", line 2, in <module>
    from .mask_renderer import *
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\insightface\app\mask_renderer.py", line 4, in <module>
    import albumentations as A
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\albumentations\__init__.py", line 5, in <module>
    from .augmentations import *
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\albumentations\augmentations\__init__.py", line 2, in <module>
    from .blur.functional import *
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\albumentations\augmentations\blur\__init__.py", line 1, in <module>
    from .functional import *
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\albumentations\augmentations\blur\functional.py", line 8, in <module>
    from albumentations.augmentations.functional import convolve
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\albumentations\augmentations\functional.py", line 11, in <module>
    from albumentations.augmentations.utils import (
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\albumentations\augmentations\utils.py", line 41, in <module>
    np.uint8: cv2.CV_8U,
              ^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'CV_8U'

2024-10-03 11:41:58,010 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui-reactor-node module for custom nodes: module 'cv2' has no attribute 'CV_8U'
2024-10-03 11:41:58,078 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-ToonCrafter\__init__.py", line 32, in <module>
    from ToonCrafter.scripts.evaluation.funcs import load_model_checkpoint, batch_ddim_sampling
  File "E:\AComfyUI/ComfyUI_windows/ComfyUI/custom_nodes/ComfyUI-ToonCrafter\ToonCrafter\scripts\evaluation\funcs.py", line 6, in <module>
    from decord import VideoReader, cpu
ModuleNotFoundError: No module named 'decord'

2024-10-03 11:41:58,078 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-ToonCrafter module for custom nodes: No module named 'decord'
2024-10-03 11:41:58,143 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-YoloWorld-EfficientSAM\__init__.py", line 1, in <module>
    from . import YOLO_WORLD_EfficientSAM
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-YoloWorld-EfficientSAM\YOLO_WORLD_EfficientSAM.py", line 6, in <module>
    import supervision as sv
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\supervision\__init__.py", line 9, in <module>
    from supervision.annotators.core import (
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\supervision\annotators\core.py", line 18, in <module>
    from supervision.draw.utils import draw_polygon
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\supervision\draw\utils.py", line 112, in <module>
    text_font: int = cv2.FONT_HERSHEY_SIMPLEX,
                     ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'FONT_HERSHEY_SIMPLEX'

2024-10-03 11:41:58,143 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-YoloWorld-EfficientSAM module for custom nodes: module 'cv2' has no attribute 'FONT_HERSHEY_SIMPLEX'
2024-10-03 11:42:00,178 - OpenGL.acceleratesupport - INFO - No OpenGL_accelerate module loaded: No module named 'OpenGL_accelerate'
2024-10-03 11:42:01,129 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui_glslnodes\__init__.py", line 9, in <module>
    from .glsl_uniforms import GlslUniforms
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui_glslnodes\glsl_uniforms.py", line 1, in <module>
    from .glsl_context import Context
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui_glslnodes\glsl_context.py", line 2, in <module>
    import moderngl
ModuleNotFoundError: No module named 'moderngl'

2024-10-03 11:42:01,130 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui_glslnodes module for custom nodes: No module named 'moderngl'
2024-10-03 11:42:01,139 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_InstantID\__init__.py", line 1, in <module>
    from .InstantID import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_InstantID\InstantID.py", line 13, in <module>
    from insightface.app import FaceAnalysis
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\insightface\__init__.py", line 18, in <module>
    from . import app
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\insightface\app\__init__.py", line 2, in <module>
    from .mask_renderer import *
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\insightface\app\mask_renderer.py", line 4, in <module>
    import albumentations as A
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\albumentations\__init__.py", line 5, in <module>
    from .augmentations import *
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\albumentations\augmentations\__init__.py", line 2, in <module>
    from .blur.functional import *
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\albumentations\augmentations\blur\__init__.py", line 1, in <module>
    from .functional import *
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\albumentations\augmentations\blur\functional.py", line 8, in <module>
    from albumentations.augmentations.functional import convolve
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\albumentations\augmentations\functional.py", line 11, in <module>
    from albumentations.augmentations.utils import (
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\albumentations\augmentations\utils.py", line 41, in <module>
    np.uint8: cv2.CV_8U,
              ^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'CV_8U'

2024-10-03 11:42:01,140 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_InstantID module for custom nodes: module 'cv2' has no attribute 'CV_8U'
2024-10-03 11:42:01,314 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_MiniCPM-V-2_6-int4\__init__.py", line 1, in <module>
    from .nodes_legacy import MiniCPM_VQA
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_MiniCPM-V-2_6-int4\nodes_legacy.py", line 6, in <module>
    from decord import VideoReader, cpu  # pip install decord
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'decord'

2024-10-03 11:42:01,314 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_MiniCPM-V-2_6-int4 module for custom nodes: No module named 'decord'
2024-10-03 11:42:04,831 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 936, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1073, in get_code
  File "<frozen importlib._bootstrap_external>", line 1130, in get_data
FileNotFoundError: [Errno 2] No such file or directory: 'E:\\AComfyUI\\ComfyUI_windows\\ComfyUI\\custom_nodes\\ComfyUI_Ultimate_SD_Upscale\\__init__.py'

2024-10-03 11:42:04,831 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_Ultimate_SD_Upscale module for custom nodes: [Errno 2] No such file or directory: 'E:\\AComfyUI\\ComfyUI_windows\\ComfyUI\\custom_nodes\\ComfyUI_Ultimate_SD_Upscale\\__init__.py'
2024-10-03 11:42:04,961 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 936, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1073, in get_code
  File "<frozen importlib._bootstrap_external>", line 1130, in get_data
FileNotFoundError: [Errno 2] No such file or directory: 'E:\\AComfyUI\\ComfyUI_windows\\ComfyUI\\custom_nodes\\FreeU\\__init__.py'

2024-10-03 11:42:04,961 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\FreeU module for custom nodes: [Errno 2] No such file or directory: 'E:\\AComfyUI\\ComfyUI_windows\\ComfyUI\\custom_nodes\\FreeU\\__init__.py'
2024-10-03 11:42:04,969 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\rembg-comfyui-node\__init__.py", line 1, in <module>
    from rembg import remove
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\rembg\__init__.py", line 5, in <module>
    from .bg import remove
  File "E:\AComfyUI\ComfyUI_windows\python_embeded\Lib\site-packages\rembg\bg.py", line 6, in <module>
    from cv2 import (
ImportError: cannot import name 'BORDER_DEFAULT' from 'cv2' (unknown location)

2024-10-03 11:42:04,969 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\rembg-comfyui-node module for custom nodes: cannot import name 'BORDER_DEFAULT' from 'cv2' (unknown location)
2024-10-03 11:42:05,848 - py.warnings - WARNING - FutureWarning: `torch.utils._pytree._register_pytree_node` is deprecated. Please use `torch.utils._pytree.register_pytree_node` instead.

2024-10-03 11:42:07,703 - root - WARNING - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\nodes.py", line 1994, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\was-node-suite-comfyui\__init__.py", line 1, in <module>
    from .WAS_Node_Suite import NODE_CLASS_MAPPINGS
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\was-node-suite-comfyui\WAS_Node_Suite.py", line 14106, in <module>
    build_info = ' '.join(cv2.getBuildInformation().split())
                          ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'getBuildInformation'

2024-10-03 11:42:07,704 - root - WARNING - Cannot import E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\was-node-suite-comfyui module for custom nodes: module 'cv2' has no attribute 'getBuildInformation'
2024-10-03 11:42:07,714 - root - INFO - 
Import times for custom nodes:
2024-10-03 11:42:07,714 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\AIGODLIKE-COMFYUI-TRANSLATION
2024-10-03 11:42:07,714 - root - INFO -    0.0 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui-ext
2024-10-03 11:42:07,714 - root - INFO -    0.0 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-CLIPSeg
2024-10-03 11:42:07,714 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\websocket_image_save.py
2024-10-03 11:42:07,714 - root - INFO -    0.0 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\FreeU
2024-10-03 11:42:07,715 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\image-resize-comfyui
2024-10-03 11:42:07,715 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_TiledIPAdapter
2024-10-03 11:42:07,715 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_ResolutionSelector
2024-10-03 11:42:07,715 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-seamless-tiling
2024-10-03 11:42:07,715 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\cg-use-everywhere
2024-10-03 11:42:07,715 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\sd-dynamic-thresholding
2024-10-03 11:42:07,715 - root - INFO -    0.0 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_Ultimate_SD_Upscale
2024-10-03 11:42:07,715 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-sampler-lcm-alternative
2024-10-03 11:42:07,715 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Image-Selector
2024-10-03 11:42:07,715 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyUI-NormalLights
2024-10-03 11:42:07,715 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Logic
2024-10-03 11:42:07,715 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\sdxl_prompt_styler
2024-10-03 11:42:07,715 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_ADV_CLIP_emb
2024-10-03 11:42:07,716 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_TiledKSampler
2024-10-03 11:42:07,716 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_IPAdapter_plus
2024-10-03 11:42:07,716 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ControlNet-LLLite-ComfyUI
2024-10-03 11:42:07,716 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\Comfyui_TTP_Toolset
2024-10-03 11:42:07,716 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-FluxExt-MZ
2024-10-03 11:42:07,716 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\efficiency-nodes-comfyui
2024-10-03 11:42:07,716 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-BRIA_AI-RMBG
2024-10-03 11:42:07,716 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui_ws
2024-10-03 11:42:07,716 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\stability-ComfyUI-nodes
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUi_String_Function_Tree
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-post-processing-nodes
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUi_NNLatentUpscale
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\masquerade-nodes-comfyui
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\As_ComfyUI_CustomNodes
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUi_PromptStylers
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_Noise
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-AutomaticCFG
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-WD14-Tagger
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\wlsh_nodes
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\rembg-comfyui-node
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\Comfyui-StableSR
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui-inpaint-nodes
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui-tooling-nodes
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-BiRefNet-ZHO
2024-10-03 11:42:07,717 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ymc-node-suite-comfyui
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_experiments
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyLiterals
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Text_Image-Composite
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\StableZero123-comfyui
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Kolors-MZ
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui_controlnet_aux
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_MiniCPM-V-2_6-int4
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui_glslnodes
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-TiledDiffusion
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-sudo-latent-upscale
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-ToonCrafter
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_FizzNodes
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_ColorMod
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyMath
2024-10-03 11:42:07,718 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui-hiforce-plugin
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui-various
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui-browser
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_UltimateSDUpscale
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfy-image-saver
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_InstantID
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Custom-Scripts
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Impact-Pack
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Frame-Interpolation
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-KwaiKolorsWrapper
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Advanced-ControlNet
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_essentials
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-KJNodes
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-YoloWorld-EfficientSAM
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-MimicMotionWrapper
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\rgthree-comfy
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\Derfuu_ComfyUI_ModdedNodes
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui_bmad_nodes
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-IC-Light
2024-10-03 11:42:07,719 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-SUPIR
2024-10-03 11:42:07,720 - root - INFO -    0.0 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui-reactor-node
2024-10-03 11:42:07,720 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui-workspace-manager
2024-10-03 11:42:07,720 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-LLaVA-Captioner
2024-10-03 11:42:07,720 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_Comfyroll_CustomNodes
2024-10-03 11:42:07,720 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-AnimateDiff-Evolved
2024-10-03 11:42:07,720 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_smZNodes
2024-10-03 11:42:07,720 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\facerestore_cf
2024-10-03 11:42:07,720 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_ExtraModels
2024-10-03 11:42:07,720 - root - INFO -    0.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Inspire-Pack
2024-10-03 11:42:07,720 - root - INFO -    0.1 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-DiffusersStableCascade
2024-10-03 11:42:07,720 - root - INFO -    0.1 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_tinyterraNodes
2024-10-03 11:42:07,720 - root - INFO -    0.1 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfy_mtb
2024-10-03 11:42:07,720 - root - INFO -    0.1 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Keyframed
2024-10-03 11:42:07,720 - root - INFO -    0.1 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\Comfyui-ergouzi-DGNJD
2024-10-03 11:42:07,720 - root - INFO -    0.1 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-BrushNet
2024-10-03 11:42:07,720 - root - INFO -    0.2 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_LayerStyle
2024-10-03 11:42:07,720 - root - INFO -    0.4 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-fastblend
2024-10-03 11:42:07,720 - root - INFO -    0.7 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Easy-Use
2024-10-03 11:42:07,720 - root - INFO -    0.7 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-BiRefNet
2024-10-03 11:42:07,721 - root - INFO -    0.8 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-LaMA-Preprocessor
2024-10-03 11:42:07,721 - root - INFO -    0.9 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\SeargeSDXL
2024-10-03 11:42:07,721 - root - INFO -    0.9 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Manager
2024-10-03 11:42:07,721 - root - INFO -    1.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_Fill-Nodes
2024-10-03 11:42:07,721 - root - INFO -    1.4 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui-art-venture
2024-10-03 11:42:07,721 - root - INFO -    1.6 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui-mixlab-nodes
2024-10-03 11:42:07,721 - root - INFO -    1.9 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\was-node-suite-comfyui
2024-10-03 11:42:07,721 - root - INFO -    1.9 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI_Custom_Nodes_AlekPet
2024-10-03 11:42:07,721 - root - INFO -    2.4 seconds (IMPORT FAILED): E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Allor
2024-10-03 11:42:07,721 - root - INFO -    3.0 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Crystools
2024-10-03 11:42:07,721 - root - INFO -    3.4 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\comfyui_segment_anything
2024-10-03 11:42:07,721 - root - INFO -    9.3 seconds: E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-Prompt-Translator
2024-10-03 11:42:07,721 - root - INFO - 
2024-10-03 11:42:07,738 - root - INFO - 

2024-10-03 11:42:07,738 - root - INFO - 

Starting server
2024-10-03 11:42:07,738 - root - INFO - [93mTo see the GUI go to: http://192.168.137.1:8188 or http://127.0.0.1:8188
2024-10-03 11:42:07,738 - root - INFO - [93mTo see the GUI go to: https://192.168.137.1:8189 or https://127.0.0.1:8189[0m
2024-10-03 11:43:36,057 - root - INFO - got prompt
2024-10-03 11:43:36,534 - root - INFO - Using pytorch attention in VAE
2024-10-03 11:43:36,535 - root - INFO - Using pytorch attention in VAE
2024-10-03 11:43:36,761 - root - ERROR - !!! Exception during processing !!! module 'cv2' has no attribute 'VideoCapture'
2024-10-03 11:43:36,762 - root - ERROR - Traceback (most recent call last):
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\execution.py", line 323, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\execution.py", line 198, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\execution.py", line 169, in _map_node_over_list
    process_inputs(input_dict, i)
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\execution.py", line 158, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 271, in load_video
    return load_video_cv(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 143, in load_video_cv
    (width, height, fps, duration, total_frames, target_frame_time) = next(gen)
                                                                      ^^^^^^^^^
  File "E:\AComfyUI\ComfyUI_windows\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 42, in cv_frame_generator
    video_cap = cv2.VideoCapture(strip_path(video))
                ^^^^^^^^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'VideoCapture'

2024-10-03 11:43:36,763 - root - INFO - Prompt executed in 0.42 seconds
```
## Attached Workflow
Please make sure that workflow does not contain any sensitive information such as API keys or passwords.
```
Workflow too large. Please manually upload the workflow from local file system.
```

## Additional Context
(Please add any additional context or steps to reproduce the error here)

**Comment by AustinMroz** [comment]
Duplicate of #185.
Th linked issue provides a potential solution [here](https://github.com/Kosinkadink/ComfyUI-VideoHelperSuite/issues/185#issuecomment-2113471677)
